README.md

```diff
@@ -135,6 +135,25 @@ If your gpu count per node is not 8, adjust:
 
 in the SBATCH command section.
 
+## Enable Float8 Training on H100s
```

Comment:
Instead of adding a long section, I wonder if it make sense to:
1. add a dedicated float8_training.md in `docs/`
2. add a fp8 training hyperlink in `key features avaible` to point to the md file inside `docs`

Reply:
sure. will move it to `docs/`